docs/enable_disable_module.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Enable and disable a Kyma module
+---
+
+## Overview
+
+Your cluster comes with the Kyma custom resource (CR) already installed. It collects all metadata about the cluster, such as enabled modules, their statuses, or synchronization, using Lifecycle Manager. Lifecycle Manager uses `moduletemplate.yaml` to enable or disable modules on your cluster. 
```

Comment:
I wouldn't mention Lifecycle Manager, @janmedrek would you agree?

Reply:
```suggestion
Your cluster comes with the Kyma custom resource (CR) already installed. It collects all metadata about the cluster, such as enabled modules, their statuses, or synchronization. The `moduletemplate.yaml` contains ModuleTemplate custom resource that is used to enable or disable modules on your cluster. 
```

I would go with something like this. 🙂